src/core/solvers/ddp.cpp

```diff
@@ -14,9 +14,10 @@ namespace crocoddyl {
 
 SolverDDP::SolverDDP(boost::shared_ptr<ShootingProblem> problem)
     : SolverAbstract(problem),
-      regfactor_(10.),
-      regmin_(1e-9),
-      regmax_(1e9),
+      reg_incfactor_(10.),
+      reg_decfactor_(10.),
+      reg_min_(1e-9),
```

Comment:
Why should it be so high? This makes totally no sense. You should be able to go to zero.

Reply:
I agree with you, reg_min should be zero.
I was planning to change it in another PR to make the review easier and focus.

However, I will change it since you agree with me ;)

Reply:
done! 
I used `1e-18` since it is the double resolution.
Additionally, I updated the log files since the numerical behavior changed a bit

Reply:
You can use `std::numeric_limits<double>::epsilon()` (from `<limits>`) or a quantity related to that (e.g. when you square/take the square root). For double, this is `2.22045e-16`.

Reply:
I agree that we should start this value from zero. It doesn't make any sense to use such ridiculously low value. 

For the reg_max, I don't understand why you decided to increase it to 1e12? 1e9 is already quite high, and if problem cannot be solved with 1e9 as the regularization, there is something wrong with the problem

Reply:
There is a parallel discussion below on the same topic. I'll duplicate my comment there instead.